CommunityToolkit.Tooling.SampleGen/ToolkitSampleMetadataGenerator.Documentation.cs

```diff
@@ -108,17 +112,19 @@ private ImmutableArray<ToolkitFrontMatter> GatherDocumentFrontMatter(SourceProdu
                 var title = ParseYamlField(ref ctx, file.Path, ref frontmatter, FrontMatterRegexTitle, "title");
                 var description = ParseYamlField(ref ctx, file.Path, ref frontmatter, FrontMatterRegexDescription, "description");
                 var keywords = ParseYamlField(ref ctx, file.Path, ref frontmatter, FrontMatterRegexKeywords, "keywords");
+
                 //// TODO: Should generate the enum from these or something?
                 var category = ParseYamlField(ref ctx, file.Path, ref frontmatter, FrontMatterRegexCategory, "category");
                 var subcategory = ParseYamlField(ref ctx, file.Path, ref frontmatter, FrontMatterRegexSubcategory, "subcategory");
+                var icon = ParseYamlField(ref ctx, file.Path, ref frontmatter, FrontMatterRegexIcon, "icon");
```

Comment:
We should make this field optional so components with existing documentation will build after merging this PR. 

Or, to leave it as required, we would need a PR for both CommunityToolkit/Windows and CommunityToolkit/Labs-Windows to update all components with this field, including the ones still in PR.

Reply:
@Arlodotexe Yeah I have a PR up Toolkit. If this gets merged in I'll update the Labs and other open PRs.